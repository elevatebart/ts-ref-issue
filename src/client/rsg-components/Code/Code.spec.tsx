import React from 'react';
import { shallow } from 'enzyme';
import { CodeRenderer } from './CodeRenderer';

describe('Code blocks', () => {
	it('should render code', () => {
		const code = '<button>OK</button>';
		const actual = shallow(<CodeRenderer classes={{}}>{code}</CodeRenderer>);

		expect(actual).toMatchSnapshot();
	});
});
