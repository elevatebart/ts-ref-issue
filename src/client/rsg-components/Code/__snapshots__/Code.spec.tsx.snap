// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Code blocks should render code 1`] = `
<code>
  &lt;button&gt;OK&lt;/button&gt;
</code>
`;
